src/test/kotlin/no/nav/familie/ef/sak/service/RevurderingServiceIntegrationTest.kt

```diff
@@ -172,6 +190,48 @@ internal class RevurderingServiceIntegrationTest : OppslagSpringRunnerTest() {
             .isEqualTo(aleneomsorgVilkårForRevurdering)
     }
 
+    @Test
+    internal fun `revurdering - skal kopiere vedtak ved satsendring`() {
+        BrukerContextUtil.clearBrukerContext()
+        val fagsakBarnetilsyn = testoppsettService.lagreFagsak(fagsak(identer = identer, stønadstype = StønadType.BARNETILSYN))
+        revurderingDto = RevurderingDto(fagsakBarnetilsyn.id, BehandlingÅrsak.SATSENDRING, kravMottatt, emptyList())
+
+        val behandling = behandling(fagsakBarnetilsyn)
+        behandlingRepository.insert(behandling)
+        val søknad = lagreSøknadForBarnetilsyn(behandling)
+        opprettVilkårForBarnetilsyn(behandling, søknad)
+        val vedtak = vedtakBarnetilsyn(
+            behandlingId = behandling.id,
+            barn = barnRepository.findByBehandlingId(behandling.id).map { it.id },
+            beløp = 8000,
+            kontantstøtteWrapper = KontantstøtteWrapper(listOf(PeriodeMedBeløp(Månedsperiode(YearMonth.of(2022, 3), YearMonth.of(2022, 4)), 1000))),
+            fom = YearMonth.of(2022, 1),
+            tom = YearMonth.of(2022, 12)
+        )
+
+        vedtakService.lagreVedtak(vedtak.tilVedtakDto(), behandling.id, StønadType.BARNETILSYN)
+
+        beregnYtelseSteg.utførSteg(saksbehandling(fagsakBarnetilsyn, behandling), vedtak.tilVedtakDto())
+        behandlingRepository.update(
+            behandling.copy(
+                status = BehandlingStatus.FERDIGSTILT,
+                resultat = BehandlingResultat.INNVILGET,
+                vedtakstidspunkt = SporbarUtils.now()
+            )
+        )
+
+        BrukerContextUtil.mockBrukerContext("Heider")
```

Comment:
Kunne vi laget en litt "lettere-å-lese" given for denne testen? 

Noe i retning av lagBarnetilsynBehandlingMedInnvilgetVedtak(perioderHer, kontantstøttePerioderHer?) 
Nå er det litt mye å lese - kanskje ikke alt like interessant?